release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -428,6 +442,16 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 [id="ocp-4-10-openshift-update-service-bug-fixes"]
 ==== OpenShift Update Service
 
+[discrete]
+[id="ocp-4-10-performance-addon-operator-bug-fixes"]
+==== Performance Addon Operator
+
+* Previously, `spec.cpu.reserved` might not be correctly set by default if `spec.cpu.isolated` is the only parameter defined in `PerformanceProfile`. You must set the settings for both `spec.cpu.reserved` and `spec.cpu.isolated` in the `PerformanceProfile`. The sets must not overlap and the sum of all CPUs mentioned must cover all CPUs expected by the workers in the target pool. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1986681[*BZ#1986681*])
```

Comment:
What is the fix/result for this? I was confused reading the doc text. Does it need a "previously" in the beginning or can that be rephrased?